src/Angor/Client/Pages/Signatures.razor

```diff
@@ -329,25 +378,28 @@
         {
             passwordComponent.ShowPassword(async () =>
             {
-                await ApproveSignature(signature); ;
+                await ApproveSignature(signature);
             });
         }
     }
 
-    private async Task ApproveSignature(SignatureRequest signature)
+
+    private async Task<OperationResult> TryApproveSignature(SignatureRequest signature)
     {
-        var operationResult = await notificationComponent.LongOperation(async () =>
+        return await notificationComponent.LongOperation(async () =>
```

Comment:
ideally we should get rid of `LongOperation` if you feel like doing it fine otherwise in the next pr